### PR TITLE
Workaround dart:html type error

### PR DESCRIPTION
**Pull request by vsmenon**
This workaround the type error in:

https://github.com/dart-lang/sdk/issues/37899

I.e., `keydown` doesn't always lead to a `KeyboardEvent` for some reason.  Need to investigate root cause further.

### CODE REVIEW

File: dwds/web/client.dart [hunk 1]
```diff
@@ -82,8 +82,8 @@ Future<void> main() {
       }
     });
 
-    window.onKeyDown.listen((e) {
-      if (const [
+    window.onKeyDown.listen((Event e) {
+      if ((e is KeyboardEvent) && const [
```
Comment (natebosch) [inline dwds/web/client.dart]: are the parens necessary here?

Comment (vsmenon) [reply dwds/web/client.dart]: No, removed.

**Comment by vsmenon** [comment]
BTW, the code size increases appears to be due to the newer SDK I'm compiling with.  I see roughly the same increase with or without the *is* check.

**Comment by jakemac53** [comment]
> BTW, the code size increases appears to be due to the newer SDK I'm compiling with. I see roughly the same increase with or without the _is_ check.

Oh ok, that makes more sense then.

**Pull request merged**